nozzle/event_handler.go

```diff
@@ -36,12 +37,34 @@ type eventHandlerImpl struct {
 	numContainerMetricReceived metrics.Counter
 }
 
+type vcapServices struct {
```

Comment:
@laullon you do not need this. Since the nozzle will be part of the tile, we will automatically set the relevant environment variables for the nozzle. No need to source the VCAP services.

Reply:
With this there is no need to seeing up env var on the tile, just link the service with the app.